sdk/storage/azure-storage-common/CMakeLists.txt

```diff
@@ -81,7 +81,7 @@ target_link_libraries(azure-storage-common PUBLIC Azure::azure-core)
 target_link_libraries(azure-storage-common INTERFACE Threads::Threads ${LIBXML2_LIBRARIES})
 
 if(MSVC)
-    target_link_libraries(azure-storage-common PRIVATE bcrypt)
+    target_link_libraries(azure-storage-common PRIVATE bcrypt Crypt32.lib)
```

Comment:
I think I wrongly changed the KEYWORDS here in PR #1212.

I need to investigate more before approving this PR.